components/Task.js

```diff
@@ -0,0 +1,27 @@
+import styled from "styled-components";
+
+export default function Task({ task }) {
+  if (!task) return;
+
+  const { _id: id, title, description, tag, createdAt } = task;
```

Comment:
```suggestion
  const {title} = task;
```

you only need the title here.

Reply:
Thanks Julia! i've changed them.